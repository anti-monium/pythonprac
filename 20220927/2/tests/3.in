8,917,510,56,40,964,703,80,66
