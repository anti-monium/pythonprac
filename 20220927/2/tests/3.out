[510, 40, 80, 703, 56, 66, 8, 917, 964]
